packages/datepicker-react/documentation/Example.tsx

```diff
@@ -3,6 +3,8 @@ import { ExampleComponentProps } from "@fremtind/jkl-portal-components";
 import { LabelVariant } from "@fremtind/jkl-core";
 import { DatePicker } from "../src";
 
+const monthsIsh = (num: number) => 1000 * 60 * 60 * 24 * (num * 30 - 5);
```

Comment:
Vil det si at du på L25-26 setter date 2 måneder tilbake i tid og 5 måneder frem i tid?

Hvorfor `- 5`?

Reply:
Det stemmer! Grunnen til at jeg ikke har satt det til et "helt" antall måneder er for å demonstrere fiksen: Buggen her har jo oppstått når du f.eks. vil gå tilbake fra 15. september til å vise august, men så er første mulige dato f.eks. 20. august. Hvis man bare setter datoen hele måneder frem/tilbake vil man aldri trigge feilen 😊